pipelines/cellranger/adapter.wdl

```diff
@@ -153,6 +151,8 @@ workflow Adapter10xCount {
   Boolean add_md5s = false
 
   String pipeline_tools_version = "v0.56.6"
+  # this is a hack to optionally force-disable the task level call-caching
+  String timestamp = "placeholder"
```

Comment:
You could write that preceding comment here instead of "placeholder".

Reply:
will do!